vite.config.ts

```diff
@@ -45,6 +45,24 @@ export default defineConfig(({ mode }) => {
                 }
             })
         ],
+        resolve: {
+            extensions: ['.mdx', '.ts', '.tsx'],
+            alias: [
+                {
+                    find: 'react/jsx-runtime',
+                    replacement: 'react/jsx-runtime.js'
+                },
+                { find: /^(react-dom\/.+)/, replacement: '$1.js' },
```

Comment:
This line has a similar bug with https://github.com/storybookjs/storybook/issues/24903#issuecomment-2118304182